CODEOWNERS

```diff
@@ -1,2 +1,3 @@
 # Default code owner
 * @PiotrMaciejKowalski
+* @KarolinaWrobel
```

Comment:
Chyba powinnam się zainteresować - co to oznacza?

Reply:
Oznacza, że twoje review będzie pozwalało na wejście do maina.

Reply:
okej